test/Microsoft.Azure.Devices.Edge.Test/Metrics.cs

```diff
@@ -41,23 +41,21 @@ class Report
             public int Failed { get; set; }
         }
 
-        void BaseConfig(EdgeConfigBuilder builder)
+        async Task Deploy(CancellationToken token)
         {
-            string metricsValidatorImage = Context.Current.MetricsValidatorImage.Expect(() => new InvalidOperationException("Missing Metrics Validator image"));
+            // First deploy different agent image. This will force agent to update environment variables
+            await this.runtime.DeployConfigurationAsync(builder => builder.GetModule("$edgeAgent").WithSettings(("image", EdgeAgentBaseImage)), token);
```

Comment:
Does this build? Looks like you have an extra closing parenthesis:
```
...WithSettings(("image", EdgeAgentBaseImage)), token)
                                             ^
```

Reply:
("image", EdgeAgentBaseImage) is a tuple